lib/mgmt_be_client.c

```diff
@@ -1068,16 +1067,8 @@ int mgmt_be_send_subscr_req(struct mgmt_be_client *client_ctx,
 	subscr_req.oper_xpaths = oper_xpaths;
 
 	/* See if we should register for notifications */
-	subscr_req.n_notif_xpaths = client_ctx->cbs.nnotify_cbs;
-	if (client_ctx->cbs.nnotify_cbs) {
-		struct mgmt_be_client_notification_cb *cb, *ecb;
-
-		cb = client_ctx->cbs.notify_cbs;
-		ecb = cb + client_ctx->cbs.nnotify_cbs;
-		for (; cb < ecb; cb++)
-			*darr_append(notif_xpaths) = cb->xpath;
```

Comment:
I'm not sure why I built this array rather than use the one it's built from; however, part of this cleanup should remove `notif_xpaths` and the `darr_free` as well.